controller/controller_server/utils.py

```diff
@@ -201,6 +227,18 @@ def generate_csi_create_snapshot_response(new_snapshot, source_volume_id):
     return res
 
 
+def generate_csi_validate_volume_capabilities_response(volume_context, volume_capabilities, parameters):
+    logger.debug("creating validate volume capabilities response")
+
+    res = csi_pb2.ValidateVolumeCapabilitiesResponse(confirmed=csi_pb2.ValidateVolumeCapabilitiesResponse.Confirmed(
+        volume_context=volume_context,
+        volume_capabilities=volume_capabilities,
+        parameters=parameters))
+
+    logger.debug("finished creating validate volume capabilities response : {0}".format(res))
```

Comment:
according to the [spec](https://github.com/container-storage-interface/spec/blob/v1.3.0/spec.md#createvolume):
```
`mount_flags` MAY contain sensitive information.
Therefore, the CO and the Plugin MUST NOT leak this information
```
therefore, we should not log the entire response (similar to why we [don't log entire requests](https://github.com/IBM/ibm-block-csi-driver/pull/191))

Reply:
its logged in every generate request..

Reply:
you mean generate response? I meant that this response is too sensitive to be logged (similar to most requests).
if you think there are more gaps like this, please specify where and why.

Reply:
why is there anything spacial here?

Reply:
the `mount_flags` is sensitive information. am I getting something wrong here?

Reply:
the `mount flag` exist in [create volume too](https://github.com/IBM/ibm-block-csi-driver/blob/c20369e8db937074241f3ee62116ce841fdd533f/controller/controller_server/utils.py#L186)

Reply:
I don't see `mount flag` in `CreateVolumeResponse`. could you link to a line which shows that it does?

Reply:
and regardless of whether or not this issue exists in other places, I think it should be fixed here

Reply:
its what you inked in first omment

Reply:
in my first comment I linked to a request, not a response.
this conversation cannot progress if you won't provide a concise line that actually shows that `CreateVolumeResponse` contains `mount_flags`...

Reply:
done